patches/server/0019-Implement-Paper-VersionChecker.patch

```diff
@@ -77,7 +79,12 @@ index 0000000000000000000000000000000000000000..1a1b50e475b9ede544b2f6d0d36632b2
 +            case -2:
 +                return Component.text("Unknown version", NamedTextColor.YELLOW);
 +            default:
-+                return Component.text("You are " + distance + " version(s) behind", NamedTextColor.YELLOW);
++                return Component.text("You are " + distance + " version(s) behind", NamedTextColor.YELLOW)
++                        .append(Component.newline())
++                        .append(Component.text("Download the new version at: ", NamedTextColor.YELLOW)
++                                .append(Component.text(DOWNLOAD_PAGE)
++                                        .hoverEvent(Component.text("Click to open", NamedTextColor.WHITE))
++                                        .clickEvent(ClickEvent.openUrl(DOWNLOAD_PAGE))));
```

Comment:
maybe use a different color like GOLD or AQUA for the link itself so it's more obvious you can click it

Reply:
Good idea, I changed it to gold